### PR TITLE
Updates for operator changes for powermax

### PR DESCRIPTION
**Pull request by shanmydell**
# Description
Updates for Operator changes for PowerMax

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/195 |

# Checklist:

- [X] Have you run a grammar and spell checks against your submission?
- [X] Have you tested the changes locally?
- [X] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by boyamurthy — APPROVED** [review]
looks good to me.

File: content/docs/csidriver/release/powermax.md [hunk 1]
```diff
@@ -6,7 +6,10 @@ description: Release notes for PowerMax CSI driver
 ## Release Notes - CSI PowerMax v2.2.0
 
 ### New Features/Changes
+- Added support for new access modes in CSI Spec 1.5.
+- Added support for PV/PVC metrics.
 - Added support for Kubernetes 1.23.
+- Added support for new access modes in CSI Spec 1.5.
```
Comment (randeepsharma) [inline content/docs/csidriver/release/powermax.md:12]: Mentioned twice.

Comment (shanmydell) [reply content/docs/csidriver/release/powermax.md]: Addressed, thanks for pointing it out @randeepsharma

**Review by santhoshhs10 — APPROVED** [review]
Looks  good to me.

**Review by boyamurthy — APPROVED** [review]

**Pull request merged**